src/App/Placement.cpp

```diff
@@ -41,7 +43,65 @@ Placement::Placement() = default;
 
 Placement::~Placement() = default;
 
+Base::Placement Placement::getPlacementFromProp(App::DocumentObject* obj, const char* propName)

```

Comment:
The purpose of App::Placement doesn't really fit to these helper functions. I would rather move them to GeoFeature or a new helper class.

Reply:
Moved to GeoFeature